person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
```

Comment:
- [ ] just as previous reviewer mentioned you should call your private method here, that is the reason you created it so this line should be ` of_age?() || @parent_permission`